modules/core/src/lib/view-manager.ts

```diff
@@ -29,12 +29,37 @@ import type View from '../views/view';
 import type {Timeline} from '@luma.gl/engine';
 import type {EventManager} from 'mjolnir.js';
 import type {ConstructorOf} from '../types/types';
-
-export default class ViewManager {
+import type MapView from '../views/map-view';
+
+export type ViewOrViews = View | View[] | null;
+type ViewStatOf<ViewT extends View> = ReturnType<ViewT['filterViewState']>;
+type OneOfViews<ViewsT extends ViewOrViews> = ViewsT extends null
+  ? MapView
+  : ViewsT extends readonly (infer ViewT)[]
+  ? ViewT
+  : ViewsT;
+export type AnyViewStateOf<ViewsT extends ViewOrViews> = ViewStatOf<OneOfViews<ViewsT>>;
+export type ViewStateMap<ViewsT extends ViewOrViews> =
+  | AnyViewStateOf<ViewsT>
+  | {[viewId: string]: AnyViewStateOf<ViewsT>};
+
+/** ViewManager props directly supplied by the user */
+type ViewManagerProps<ViewsT extends ViewOrViews> = {
+  views: ViewsT;
+  viewState: ViewStateMap<ViewsT>;
+  onViewStateChange?: (
+    params: ViewStateChangeParameters<AnyViewStateOf<ViewsT>> & {viewId: string}
+  ) => void;
+  onInteractionStateChange?: (state: InteractionState) => void;
```

Comment:
Is this a relatively new user prop? I only recognize the others as also DeckProps

Reply:
No, it has been supported since v8.